app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WidgetManagerDelegate.java

```diff
@@ -25,8 +27,9 @@
     void removeUpdateListener(@NonNull UpdateListener aUpdateListener);
     void pushBackHandler(@NonNull Runnable aRunnable);
     void popBackHandler(@NonNull Runnable aRunnable);
-    void fadeOutWorld();
-    void fadeInWorld();
+    void pushWorldBrightness();
+    void setWorldBrightness(float aBrightness);
```

Comment:
Do we need separate `setWorldBrightness()` and `pushWorldBrightness()` functions? Are there instances where we want to set the brightness and not have it push the current state?

Reply:
all the cases should call `setWorldBrightness` after `pushWorldBrightness`. So we can add the parameter to setWorldBrightness. I'll update the PR